docs/contentSharing.md

```diff
@@ -35,7 +35,7 @@ In order for another course to use questions from your course into their assessm
 
 Questions can either be shared publicly so that anyone can preview the questions and use them in their course, or shared only to specific other courses using sharing sets.
 
-Any question that is marked with `"sharePublicly": true` or `"shareSourcePublicly": true` will be considered and displayed as being published for free use under the [CC-BY-NC license](https://www.creativecommons.org/licenses/by-nc/4.0/).
+Any question that is marked with `"sharePublicly": true` or `"shareSourcePublicly": true` will be considered and displayed as being published for free use under the [CC-BY-NC license](https://www.creativecommons.org/licenses/by-nc/4.0/). We recommend adding an `authors` field to the [metadata](question/index.md#metadata-infojson) of shared questions to allow attribution of the original author(s).
```

Comment:
Add an example config for authors to this page.

Reply:
Better comment would be that a sample config would be great to have somewhere in the documentation, it doesn't have to go exactly here.